Main cleanup of Muon v 6.3 release notes

sf1919 opened the pull request:
**Description of work.**
The main refactoring the the Muon release notes ahead of release of 6.3. Includes:-

- Removing unused headings and developer warning text
- Reordering of notes where appropriate
- Rewording notes to improve consistency
- Adding links
- Spelling & Grammar check
- Adding an image for new error shading feature

Please note that the Model Fitting notes at the end are currently invisible to users. Apart from moving some notes into this section and rewording all to Model Fitting for consistency no other editing has taken place. This is because we are awaiting approval for release of this feature. These notes will either be revamped or moved to v.6.4 release notes before release.

**To test:**

-     remove build/docs/doctrees
-     build the documents again and check there are no warnings
-     Check that the links work
-     Check the information is still correct
-     Are the images visible and clearly illustrate the new feature/improvement?
-     Grammar/Spelling check

*There is no associated issue.*

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

sf1919 committed:
Edited Muon release notes and added image

- Tidied the release notes including spelling changes, adding links, reorganising and rewording to make more consistent.
- Removed unused headings
- Added image for Muon error shading feature

sf1919 committed:
Additional changes to v6.3 Muon release notes

- changed error shading image
- fixed broken fitting function links
- reformatted text to highlight specific concepts (e.g. alpha)

AnthonyLim23 commented on docs/source/release/v6.3.0/muon.rst:
```diff
@@ -5,77 +5,67 @@ Muon Changes
 ############
 
-- Instead of plotting the confidence interval of a fit as an error bar, it is now represented by a shaded region.
+- **Instead of plotting the confidence interval of a fit as an error bar, it is now represented by a shaded region.**
```
is this supposed to be in bold?

sf1919 replied:
Any note that accompanies an image is supposed to be in bold.

AnthonyLim23 commented on docs/source/release/v6.3.0/muon.rst:
```diff
@@ -5,77 +5,67 @@ Muon Changes
-- The numerical values in the `run info` box on the home tab are now rounded to either 4 significant figures or a whole number, whichever is more precise.
+- Changes have been made to improve the speed of Muon Analysis and Frequency Domain Analysis.
+- The :ref:`Results Tab <muon_results_tab-ref>` will now dispay a warning (red text and a tooltip) if the results table already exists.
 - The results table now produces errors for log values (when they are available).
```
is it deliberate that above has a link for the results tab and this does not?

sf1919 replied:
Are the results table and the result tab the same thing? I wasn't sure.

AnthonyLim23 replied:
ah no, sorry I misread it (missed the le)

AnthonyLim23 commented on docs/source/release/v6.3.0/muon.rst:
```diff
@@ -85,35 +75,32 @@ Improvements
-  * `ActivationK <func-ActivationK>` can be used for data in Kelvins.
-  * `ActivationmeV <func-ActivationmeV>`can be used for data in meV.
+  * :ref:`ActivationK <func-ActivationK>` can be used for data in Kelvins.
+  * :ref:`ActivationmeV <func-ActivationmeV>`can be used for data in meV.
```
This link does not work

sf1919 replied:
Is it both links that don't work or just the second one?

AnthonyLim23 commented on docs/source/release/v6.3.0/muon.rst:
```diff
@@ -85,35 +75,32 @@ Improvements
-  * `DecoupAsymPowderMagLong <func-DecoupAsymPowderMagLong>` can be used for longitudinal polarization.
-  * `DecoupAsymPowderMagRot <func-DecoupAsymPowderMagRot>`can be used for rotational asymmetry.
+  * :ref:`DecoupAsymPowderMagLong <func-DecoupAsymPowderMagLong>` can be used for longitudinal polarization.
+  * :ref:`DecoupAsymPowderMagRot <func-DecoupAsymPowderMagRot>`can be used for rotational asymmetry.
```
This link does not work

sf1919 replied:
Again is this both links or just the second one?

AnthonyLim23 replied:
the second in both cases

AnthonyLim23 replied:
I think its just a formatting problem

sf1919 replied:
Yeah its because there is no space after the ` at the end of the link and the next word.

AnthonyLim23 commented on docs/source/release/v6.3.0/muon.rst:
```diff
@@ -85,35 +75,32 @@ Improvements
-* Added a :ref:`Muonium-style Decoupling Curve <func-MuoniumDecouplingCurve>` function to MuonModelling Fit Functions.
-* Added a :ref:`Power Law <func-PowerLaw>` fitting function to MuonModelling Fit Functions.
-* Added a ref:`Smooth Transition <func-SmoothTransition>` function to MuonModelling Fit Functions.
+* Added a :ref:`Magentic Order Parameter<func-MagneticOrderParameter>` function to ``MuonModelling\Magentism`` Fit Functions.
```
I followed the link and the equation is not formatted correctly

AnthonyLim23 replied:
also A0 should be A_0

sf1919 committed:
Review corrections

- reformatted unusable links
- reformatted equation in Magnetic Order Parameter Fitting Function documentation

AnthonyLim23 commented:
the decoupling functions have mangled equations in their docs

sf1919 committed:
Fixed formatting in new Muon Fitting Function docs

AnthonyLim23 reviewed (approved):
Looks good to me
